storage/src/main/java/io/atomix/storage/buffer/ByteBufferBytes.java

```diff
@@ -120,6 +120,15 @@ public Bytes read(int position, byte[] bytes, int offset, int length) {
     return this;
   }
 
+  @Override
+  public Bytes read(final int offset, final ByteBuffer dst, final int dstOffset, final int length) {
+    for (int i = 0; i < length; i++) {
+      dst.put(dstOffset + i, (byte)readByte(offset + i));
```

Comment:
doesn't this functionality already exist in the ByteBuffer?

Reply:
I don't think so? I'm not sure what you're referring to. `Bytes` is an Atomix class, so there's no method from `ByteBuffer` that can read from it?

Reply:
Hm but it uses internally the byteBuffer?

Reply:
`Bytes`? It can be either `byte[]`, `ByteBuffer`, `FileBytes`, etc.